src/3rdparty/mjcpy2/mujoco_osg_viewer.cpp

```diff
@@ -478,7 +478,7 @@ void MujocoOSGViewer::SetData(const mjData* d) {
 
 void NewModelFromXML(const char* filename,mjModel*& model) {
     char errmsg[100];
-    model = mj_loadXML(filename, errmsg);
+    model = mj_loadXML(filename, NULL, errmsg, sizeof(errmsg));
```

Comment:
Not sure what to do about this one. If we end up adding that cmake flag, could use it here to choose the appropriate function signature.